manifests/q/Qingfeng/HeyboxWow/1.9.12/Qingfeng.HeyboxWow.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.4 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Qingfeng.HeyboxWow
+PackageVersion: 1.9.12
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  Silent: update
+  SilentWithProgress: update
+UpgradeBehavior: install
+ReleaseDate: 2023-03-03
+Installers:
+- Architecture: x64
+  InstallerUrl: https://macc.max-c.com/wow/wow_installer_1.9.12.1.exe
```

Comment:
Hi @SpecterShell,

I wasn't able to find this InstallerUrl from the PackageUrl. Is the link somewhere else on the developer's website?

Reply:
From their API:

```PowerShell
(Invoke-RestMethod -Uri 'https://accoriapi.xiaoheihe.cn/wow/check_new_version_v2/' -Method Post).result.version_list
```

Reply:
@stephengillie 👀

Reply:
Oh - sorry. A POST call to the developer's API isn't quite a location on their website. The PackageUrl is really meant to be a GET call and not a POST, otherwise we might replace it with the API call. I'm not exactly sure how to proceed.